### PR TITLE
Adds Japanese translation

### PR DESCRIPTION
**Pull request by KoheiKanagu**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?
I did the translation into Japanese.

## Did you add any dependencies?
No

## How did you test the change?
N/A

### CODE REVIEW

**Comment by suztomo** [comment]
```

Analyzing flutter...                                            
8

9
   info • This class (or a class which this class inherits from) is marked as '@immutable', but one or more of its instance fields are not final: CarouselView.pageController • lib/carousel_page.dart:48:7 • must_be_immutable
10
   info • Unnecessary new keyword • lib/carousel_page.dart:55:35 • unnecessary_new
11

12
2 issues found. (ran in 13.5s)

```

File: client/flutter/lib/generated/intl/messages_ja.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
+// This is a library that provides messages for a zh locale. All the
```
Comment (suztomo) [inline client/flutter/lib/generated/intl/messages_ja.dart]: zh locale? Strange.

File: client/flutter/lib/l10n/intl_ja.arb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "learnMore": "もっと詳しく",
+    "protectYourself": "自分を守る",
+    "checkYourHealth": "あなたの健康をチェックする",
+    "localMaps": "ローカルマップ",
+    "shareTheApp": "アプリを共有する",
+    "healthCheck": "ヘルスチェック",
+    "feelingDistressed": "悩んでいますか？",
+    "washHands": "石鹸と流水でこまめに手を洗う",
+    "cougningAndSneezing": "咳やくしゃみをするときは、曲げた肘やティッシュで口と鼻を覆う",
+    "throwAwayTissue": "使用後すぐに蓋を閉じたゴミ箱にティッシュを捨てる",
```
Comment (suztomo) [inline client/flutter/lib/l10n/intl_ja.arb]: `蓋がついたゴミ箱`

File: client/flutter/lib/generated/intl/messages_ja.dart (hunk 1 above)
Comment (KoheiKanagu) [reply client/flutter/lib/generated/intl/messages_ja.dart]: @suztomo Thank you! Fixed

File: client/flutter/lib/l10n/intl_ja.arb (hunk 2 above)
Comment (KoheiKanagu) [reply client/flutter/lib/l10n/intl_ja.arb]: @suztomo Thank you! Fixed

**Comment by sapte91** [comment]
@KoheiKanagu Thank you so much for your support of this project and for the time you spent on this PR - we’re so humbled by your willingness to jump in! We have an important update on localization for the application: We’ll primarily be sourcing and managing translations from Lingo24, our translation management system. Also, our designs changed, so these translations are now out of date. As such, I’m closing this PR.

We will still need translation volunteers - as our set of supported languages and locales grows, there will be plenty of opportunity to contribute! Our projected needs for the next two months are in native speaker LQA and translations for languages that we cannot support on Lingo24. We apologize that we are still in the early process of figuring out how to manage all the contributions and will have more details soon. Until then, please fill out the volunteer form below and we’ll get back in touch shortly.

https://forms.gle/DTqvaqh4mE7bDe5U7

**Pull request closed without merging**